code/controllers/subsystem/vis_overlays.dm

```diff
@@ -5,12 +5,10 @@ SUBSYSTEM_DEF(vis_overlays)
 	init_order = INIT_ORDER_VIS
 
 	var/list/vis_overlay_cache
-	var/list/unique_vis_overlays
```

Comment:
What's going on here ?

Reply:
The list is totally unused, and just hangs refs